docs/conf.py

```diff
@@ -30,7 +32,7 @@
 from docutils.utils import get_source_line
 
 
-def _warn_node(self, msg, node):
+def _warn_node(self, msg, node, *args, **kwargs):
```

Comment:
This was suddenly necessary because of this line in sphinx: https://github.com/sphinx-doc/sphinx/blob/1fbc0c9a1611e343e1386daa01c6229ffeab7555/sphinx/environment.py#L917

Perhaps we're missing an error that didn't occur before.